plugins/modules/aci_system_banner.py

```diff
@@ -0,0 +1,297 @@
+    type: str
+  gui_alias:
+    description:
+    - The system GUI alias.
+    type: str
+  controller_banner:
+    description:
+    - The contents of the CLI informational banner to be displayed before user login authentication.
+    - The CLI banner is a text based string printed as-is to the console.
+    type: str
+  switch_banner:
+    description:
+    - The switch banner message.
+    type: str
+  application_banner:
+    description:
+    - The application banner message.
+    type: str
+  severity:
+    description:
+    - The application banner severity.
+    type: str
+    choices: [ critical, info, major, minor, warning ]
+  gui_banner:
+    description:
+    - The contents of the GUI informational banner to be displayed before user login authentication.
+    - When I(gui_banner) starts with I(http://) or I(https://) the banner will be of URL type.
+    - Note that the URL site owner must allow the site to be placed in an iFrame to display the informational banner.
+    type: str
+  state:
+    description:
+    - Use C(present) for updating.
+    - Use C(query) for listing an object.
+    type: str
+    choices: [ present, query ]
+    default: present
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+seealso:
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(aaa:PreLoginBanner).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+
+author:
+- Tim Cragg (@timcragg)
+- Akini Ross (@akinross)
+"""
+
+EXAMPLES = r"""
+- name: Configure banner
+  cisco.aci.aci_system_banner:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    gui_alias: "Test GUI Alias"
+    controller_banner: "Test Controller Banner"
+    application_banner: "Test Application Banner"
+    severity: critical
+    switch_banner: "Test Switch Banner"
+    gui_banner: "Test GUI Banner"
+    state: present
+  delegate_to: localhost
+
+- name: Configure banner with a url
+  cisco.aci.aci_system_banner:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    gui_alias: "Test GUI Alias"
+    controller_banner: "Test Controller Banner Message"
+    application_banner: "Test Application Banner"
+    severity: critical
+    switch_banner: "Test Switch Banner Message"
+    gui_banner: https://www.cisco.com
+    state: present
+  delegate_to: localhost
+
+- name: Query banner
+  cisco.aci.aci_system_banner:
+    host: apic
+    username: admin
+    password: SomeSecretPassword
+    name: ans_remote_path
+    state: query
+  delegate_to: localhost
+"""
+
+RETURN = r"""
+current:
+  description: The existing configuration from the APIC after the module has finished
+  returned: success
+  type: list
+  sample:
+    [
+        {
+            "fvTenant": {
```

Comment:
I guess the other modules also just use fvTenant here as examples. Is this standard or have we not changed them?

Reply:
This is a section that reflects the [return block](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block) format with the same sample values used in all modules. I think we should just keep fvTenant since this has been done consistently throughout the repo and it is only to demonstrate the format of the return.